config_builder.go

```diff
@@ -113,3 +128,176 @@ func (d *DefaultWalletImpl) ValidateMacaroon(ctx context.Context,
 func (d *DefaultWalletImpl) Permissions() map[string][]bakery.Op {
 	return nil
 }
+
+// DatabaseInstances is a struct that holds all instances to the actual
+// databases that are used in lnd.
+type DatabaseInstances struct {
```

Comment:
It looks like the invoice registry is still using the main `*channeldb.DB` instance. Taking that one apart into clearly separated DBs will be a bigger task I think, mainly because we need a new concept for schema migrations (see comment above).